File: C#/Program19.cs
/*

Write a program which accept number from user and display its digits in reverse 
order.
Input : 2395 
Output : 
5
9
3
2

*/

using System;

class Demo{
 
    public void DisplayDigitReverse(int iNo){
     
        int iDigit  = 0;

        while(iNo != 0){

            iDigit = iNo % 10;

            Console.WriteLine(iDigit);

            iNo = iNo / 10;

        }


    }

}

class program19{
       
     public static void Main(string[] args){

            int iNo = Convert.ToInt32(Console.ReadLine());
            
            Demo dobj = new Demo();

            dobj.DisplayDigitReverse(iNo);
            

     }
     
}

File: JAVA/Program19.java
/*

Accept number from user and display below pattern.
Input : 5
Output : 5 # 4 # 3 # 2 # 1 #

*/


import java.util.*;

class Demo19
{
	 public void Display(int iNo)
	 {
	 	int i = 0;

         System.out.print("\n");

	 	for(i = iNo; i >= 1; i--)
	 	{
	 		System.out.print(i+" # \t");
	 		
	 	}
	 }

}

class Program19
{
	public static void main(String[] args) 
	{
	   Scanner sobj = new Scanner(System.in);
	   Demo19 dobj = new Demo19();

	   int iVal = 0;
       
       System.out.print("\nEnter Number : ");
       iVal = sobj.nextInt();

	   dobj.Display(iVal);

	   System.out.print("\n");

	}
}